### ISSUE TITLE
Full replays from Olympics.com seems to be broken

### ISSUE THREAD

**Issue by Zalbu** [yt-dlp/yt-dlp#2896]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.02.04**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Sweden

### Description

Trying to download full replays from Beijing 2022 doesn't work, but it works without problem when downloading Tokyo 2020 replays. The URLs are different compared to earlier events, the URLs that don't work are `https://olympics.com/en/original-series/episode/` when the previous URLs that work are `https://olympics.com/en/video/`

### Verbose log

```shell
[debug] Command-line config: ['-vU', '-F', 'https://olympics.com/en/original-series/episode/mixed-relay-biathlon-beijing-2022-replays']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2022.02.04 [c1653e9] (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.22000-SP0
[debug] exe versions: ffmpeg git-2020-02-03-1c15111, ffprobe git-2020-02-03-1c15111
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
Latest version: 2022.02.04, Current version: 2022.02.04
yt-dlp is up to date (2022.02.04)
[debug] [generic] Extracting URL: https://olympics.com/en/original-series/episode/mixed-relay-biathlon-beijing-2022-replays
[generic] mixed-relay-biathlon-beijing-2022-replays: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] mixed-relay-biathlon-beijing-2022-replays: Downloading webpage
[generic] mixed-relay-biathlon-beijing-2022-replays: Extracting information
[debug] Looking for video embeds
[debug] Identified a JSON LD
[generic] mixed-relay-biathlon-beijing-2022-replays: Downloading m3u8 information
ERROR: [generic] Failed to download m3u8 information: HTTP Error 403: Forbidden (caused by <HTTPError 403: 'Forbidden'>); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U (caused by <HTTPError 403: 'Forbidden'>); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using -U
  File "yt_dlp\extractor\common.py", line 730, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3558, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default
```

**Comment by pukkandan** [comment]
@Ashish0804 @u-spec-png

**Comment by tampa-bay** [comment]
Looks like a situation where the video exists at both the original and this address, as they both call the same api endpoint with the same parameters.  @Zalbu can you get to the replay from https://olympics.com/en/video/mixed-relay-biathlon-beijing-2022-replays ?

As a short-term fix, until someone who knows what they are doing touches it you can change line 12 in extractors/olympics.py
 
from: 
_VALID_URL = r'https?://(?:www\.)?olympics\.com(?:/tokyo-2020)?/[a-z]{2}/(?:replay|video)/(?P<id>[^/#&?]+)'

to:
_VALID_URL = r'https?://(?:www\.)?olympics\.com(?:/tokyo-2020)?/[a-z]{2}/(?:replay|video|original-series/episode)/(?P<id>[^/#&?]+)'

**Comment by Zalbu** [comment]
> Looks like a situation where the video exists at both the original and this address, as they both call the same api endpoint with the same parameters. @Zalbu can you get to the replay from https://olympics.com/en/video/mixed-relay-biathlon-beijing-2022-replays ?

Yup, both accessing it in the browser and downloading it with ytp-dl works like normal, how strange. And now I get sent to the correct `https://olympics.com/en/video/` URL as well when accessing the video from the Olympics website. 

Shouldn't be a difficult fix to patch then if only another supported URL needs to be added.

**Comment by nhoizey** [comment]
Looks like replays from Paris 2024 Olympics are also broken, for example with https://olympics.com/fr/paris-2024/videos/jpn-fra-phase-de-groupe-h-basketball-jeux-olympiques-de-paris-2024

```log
[generic] Extracting URL: https://olympics.com/fr/paris-2024/videos/jpn-fra-phase-de-groupe-h-basketball-jeux-olympiques-de-paris-2024
[generic] jpn-fra-phase-de-groupe-h-basketball-jeux-olympiques-de-paris-2024: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] jpn-fra-phase-de-groupe-h-basketball-jeux-olympiques-de-paris-2024: Extracting information
ERROR: Unsupported URL: https://olympics.com/fr/paris-2024/videos/jpn-fra-phase-de-groupe-h-basketball-jeux-olympiques-de-paris-2024
```

**Comment by bashonly** [comment]
Fixed in 2f1ddfe12a2c174bc777264c5c8ffe7ca0922d94
